style.css

```diff
@@ -80,4 +82,143 @@
     flex-direction: row;
     list-style-type: none;
   }
+
+  .works {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-content: flex-end;
+    align-items: center;
+    padding: 4rem;
+  }
+
+  .recent {
+    width: 327px;
+    height: 83px;
+    font-family: 'Crete Round', serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+  }
+
+  .gridcontainer1 {
+    display: grid;
+  }
+
+  .stories {
+    font-family: 'Crete Round', serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 32px;
+    line-height: 44px;
+    color: #091e42;
+  }
+
+  .selection {
+    font-family: 'Inter', serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #091e42;
+  }
+
+  .listbutton {
+    list-style: none;
+    display: flex;
+    gap: 12px;
+    margin-top: 1rem;
+  }
+
+  .button1,
+  .button2,
+  .button3,
+  .button4 {
+    border: 1px solid #8993a4;
+    padding: 10px 12px;
+  }
+
+  .button5 {
+    background: #ff6b00;
+    padding: 12px;
+    margin-top: 12px;
+    width: 124px;
+    height: 48px;
+    color: #fff;
+  }
+
+  .listproject {
+    background-image: url("secondimage.png");
+    background-repeat: no-repeat;
+    margin-top: 2rem;
+    height: 50vh;
+  }
```

Comment:
Nice work on using CSS Grid for the layout of the main container :star_struck:. Kindly use flexbox for the layout of the individual container/card also. This is very important since it will help make your project more flexible. :thinking: 

Kindly fix the space created below the `see project` button. This is not found on the figma design. 
You can also consider removing the border around the buttons 

Figma design

![Screenshot from 2022-09-25 00-29-24](https://user-images.githubusercontent.com/65233860/192123300-ddbcd1c9-4f0b-4bdc-8820-6afe6b2b1edb.png)

Your design

![Screenshot from 2022-09-25 00-29-36](https://user-images.githubusercontent.com/65233860/192123307-6de99666-c637-4192-bb59-86272447c7ce.png)